### PR TITLE
CSV file verification information addition

### PR DESCRIPTION
**Pull request by joinimran**
As mentioned by the user, I added a note regarding the CSV file. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5510

### CODE REVIEW

File: windows/deployment/windows-autopilot/add-devices.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ Note that the hardware hash also contains details about when it was generated, s
 
 Starting with System Center Configuration Manager current branch version 1802, the hardware hashes for existing Windows 10 version 1703 and higher devices are automatically collected by Configuration Manager. See the [What’s new in version 1802](https://docs.microsoft.com/sccm/core/plan-design/changes/whats-new-in-version-1802#report-on-windows-autopilot-device-information) documentation for more details.  The hash information can be extracted from Configuration Manager into a CSV file.
 
+>[!Note]
```
Comment (illfated) [inline windows/deployment/windows-autopilot/add-devices.md]: Please add MD indent compatibility spacing:
```suggestion
> [!Note]
```

File: windows/deployment/windows-autopilot/add-devices.md [hunk 2]
```diff
@@ -63,6 +63,9 @@ Note that the hardware hash also contains details about when it was generated, s
...
+>Before uploading the CSV file on intune, please make sure that 1st row contains Device Serial Number,Windows Product ID,Hardware Hash,Group Tag,Assigned User. If there is header information on the top of CSV file, please delete that header information. See details at [Enroll Windows devices in Intune](https://docs.microsoft.com/intune/enrollment/enrollment-autopilot).
```
Comment (illfated) [inline windows/deployment/windows-autopilot/add-devices.md]: more spacing needed:
```suggestion
> Before uploading the CSV file on intune, please make sure that 1st row contains Device Serial Number, Windows Product ID, Hardware Hash, Group Tag, Assigned User. If there is any header information on the top of CSV file, please delete that header information. See details at [Enroll Windows devices in Intune](https://docs.microsoft.com/intune/enrollment/enrollment-autopilot).
```

**Comment by jvsam** [comment]
@JohanFreelancer9 for copy review. Thank you.

File: windows/deployment/windows-autopilot/add-devices.md (hunk 2 above)
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/add-devices.md]:
```suggestion
> Before uploading the CSV file on Intune, please make sure that the first row contains the device serial number, Windows product ID, hardware hash, group tag, and assigned user. If there is header information on the top of CSV file, please delete that header information. See details at [Enroll Windows devices in Intune](https://docs.microsoft.com/intune/enrollment/enrollment-autopilot).
```

**Comment by joinimran** [comment]
Ready to go live.

**Comment by illfated** [comment]
@JohanFreelancer9 : it looks like your suggestion got marked as resolved without those changes being committed, except for the spacing from my suggestion. Are you OK with this decision?

**Comment by JohanFreelancer9** [comment]
@illfated - Not really. Here's why:
1. Intune is a product name and should be capitalized.
2. Always spell out ordinal numbers (1st) in accordance with the Microsoft Style Guide.
3. listed items should not be capitalized (refer to https://docs.microsoft.com/intune/enrollment/enrollment-autopilot where the same items are listed.

File: windows/deployment/windows-autopilot/add-devices.md [hunk 3]
```diff
@@ -63,6 +63,9 @@ Note that the hardware hash also contains details about when it was generated, s
 
 Starting with System Center Configuration Manager current branch version 1802, the hardware hashes for existing Windows 10 version 1703 and higher devices are automatically collected by Configuration Manager. See the [What’s new in version 1802](https://docs.microsoft.com/sccm/core/plan-design/changes/whats-new-in-version-1802#report-on-windows-autopilot-device-information) documentation for more details.  The hash information can be extracted from Configuration Manager into a CSV file.
 
+> [!Note]
+> Before uploading the CSV file on intune, please make sure that 1st row contains Device Serial Number, Windows Product ID, Hardware Hash, Group Tag, Assigned User. If there is any header information on the top of CSV file, please delete that header information. See details at [Enroll Windows devices in Intune](https://docs.microsoft.com/intune/enrollment/enrollment-autopilot).
```
Comment (illfated) [inline windows/deployment/windows-autopilot/add-devices.md]:
```suggestion
> Before uploading the CSV file on Intune, please make sure that the first row contains the device serial number, Windows product ID, hardware hash, group tag, and assigned user. If there is header information on the top of CSV file, please delete that header information. See details at [Enroll Windows devices in Intune](https://docs.microsoft.com/intune/enrollment/enrollment-autopilot).
```

**Comment by joinimran** [comment]
Suggestion applied. Thanks @JohanFreelancer9 . When I have applied previous commits the last commits show as outdated and I looked for the details where it was pointing to language locale which was then removed by editing the file and commit that patch in current PR. 

We are all set now and ready to go live.

**Comment by jvsam** [comment]
Hi @greg-lindsay for your review and approval. Please merge if you see no issues with this PR.

Happy holidays!

**Pull request merged**